src/pages/api/graphql.tsx

```diff
@@ -62,36 +53,36 @@ const clientConnection = async () => {
   }
 }
 
-export const apolloServer = new ApolloServer({
+export const apolloServer = new ApolloServer<ServerContext>({
   typeDefs,
   resolvers,
-  dataSources: () => {
-    return {
-      keystoneAPI: new KeystoneAPI(),
-      weatherAPI: new WeatherAPI(),
-      personnelAPI: new PersonnelAPI(),
-    }
-  },
+  status400ForVariableCoercionErrors: true,
   cache: 'bounded',
   plugins: [ApolloServerPluginLandingPageDisabled()],
-  context: async ({ req, res }) => {
+})
+
+export default startServerAndCreateNextHandler(apolloServer, {
+  context: async (req, res) => {
     const session = await getSession(req, res)
+    const { cache } = apolloServer
```

Comment:
Again, not sure if we need the cache yet in our data sources, but this is where it gets passed in, so we pull it from the server object.